databroker/src/grpc/kuksa_val_v2/val.rs

```diff
@@ -1039,13 +928,189 @@ mod tests {
         PublishValuesRequest, SignalId, Value,
     };
 
+    // Helper for adding an int8 signal and adding value
+    async fn helper_add_int8(
+        broker: &DataBroker,
+        name: &str,
+        value: i32,
+        timestamp: std::time::SystemTime,
+    ) -> Result<i32, Vec<(i32, UpdateError)>> {
+        let authorized_access = broker.authorized_access(&permissions::ALLOW_ALL);
+        let entry_id = authorized_access
+            .add_entry(
+                name.to_owned(),
+                broker::DataType::Int8,
+                broker::ChangeType::OnChange,
+                broker::EntryType::Sensor,
+                "Some Description That Does Not Matter".to_owned(),
+                Some(broker::types::DataValue::Int32(-5)), // min
+                Some(broker::types::DataValue::Int32(10)), // max
+                None,
+                None,
+            )
+            .await
+            .unwrap();
+
+        match authorized_access
+            .update_entries([(
+                entry_id,
+                broker::EntryUpdate {
+                    path: None,
+                    datapoint: Some(broker::Datapoint {
+                        //ts: std::time::SystemTime::now(),
+                        ts: timestamp,
+                        source_ts: None,
+                        value: broker::types::DataValue::Int32(value),
+                    }),
+                    actuator_target: None,
+                    entry_type: None,
+                    data_type: None,
+                    description: None,
+                    allowed: None,
+                    min: None,
+                    max: None,
+                    unit: None,
+                },
+            )])
+            .await
+        {
+            Ok(_) => Ok(entry_id),
+            Err(details) => Err(details),
+        }
+    }
+
+    #[tokio::test]
+    async fn test_update_entries_min_max_int8() {
+        let broker = DataBroker::default();
+
+        let timestamp = std::time::SystemTime::now();
+
+        match helper_add_int8(&broker, "test.datapoint1", -6, timestamp).await {
```

Comment:
What about adding also `min` and `max` as parameters to this `helper_add_...` functions to make it easier to understand at first glance what should happen here? I often had to look the `min` and `max` values used above.

Reply:
As of today all helper functions have "predefined" max/min-values, as I did not had any big need to let the limits vary. I could add min and max as args, but unless we introduce some more advanced argument builder it will just result in3 magic numbers unless your IDE can shed light on what each position means.

Please let me know if you want use to add them as arguments.


`helper_add_int8(&broker, "test.datapoint1", -10, 5, -6, timestamp).await `